superset-frontend/src/components/Chart/DrillBy/DrillByModal.tsx

```diff
@@ -127,7 +127,6 @@ export default function DrillByModal({
 }: DrillByModalProps) {
   const theme = useTheme();
 
-  console.log('DUPA', formData);
```

Comment:
![image](https://user-images.githubusercontent.com/33317356/231785658-ab4ea895-7ec8-4100-bc7b-6f09c2e5d691.png)
Not like I've ever done anything similar 😆😆😆